config/checkstyle/checkstyle-suppressions.xml

```diff
@@ -5,4 +5,5 @@
 
 <suppressions>
     <suppress checks="[a-zA-Z0-9]*" files="[\\/]grammar[\\/]" />
+    <suppress files="[\\/]generated-src[\\/]" checks="[a-zA-Z0-9]*"/>
```

Comment:
were we checking generated code before? I assume we were not, but maybe this is in a new location? It would seem odd to me to have more than one place for checkstyle suppression